rmgpy/data/base.py

```diff
@@ -295,13 +295,26 @@ def get_species(self, path, resonance=True):
         Load the dictionary containing all of the species in a kinetics library or depository.
         """
         from rmgpy.species import Species
+        from rmgpy.molecule.fragment import Fragment
+        import re
```

Comment:
Check these ones if you have time, since this file is not cythonized, but otherwise ignore this comment

Reply:
Done! Just moved